docs-v2/pages/connect/_meta.tsx

```diff
@@ -3,37 +3,40 @@ export default {
     "title": "Overview",
   },
   "use-cases": {
-    "title": "Use Cases",
+    "title": "Use cases",
   },
   "quickstart": {
     "title": "Quickstart",
   },
   "workflows": {
     "title": "Running workflows",
   },
+  "components": {
+    "title": "Running components",
+  },
   "api": {
-    "title": "API & SDK Reference",
+    "title": "API & SDK reference",
   },
   "tokens": {
-    "title": "Connect Tokens",
+    "title": "Connect tokens",
   },
   "environments": {
     "title": "Environments",
   },
   "oauth-clients": {
-    "title": "OAuth Clients",
+    "title": "OAuth clients",
   },
   "webhooks": {
     "title": "Webhooks",
   },
   "connect-link": {
-    "title": "Connect Link",
+    "title": "Connect link",
```

Comment:
Sorry this is actually the only one we should keep since "Connect Link" is the proper name of the thing!